docs/pymc_aesara.ipynb

```diff
@@ -0,0 +1,1819 @@
+{
```

Comment:
<p>Maybe explain more what can be seen here. Like Elemwise, composite? InplaceDimShuffle? Just to give some intuition.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Or perhaps not dprint the function</p>

Reply:
<p>I will remove the <code>dprint</code>.</p>